cmd/snap-bootstrap/cmd_create_partitions_test.go

```diff
@@ -59,6 +62,37 @@ func (s *cmdSuite) TestCreatePartitionsMount(c *C) {
 	c.Assert(n, Equals, 1)
 }
 
+var testModel = `type: model
```

Comment:
Same comment as aobe, the entire file cmd_create_partitions_test.go will go away

Reply:
Samuele answered this, want this and a spread test that measures the model